connectivity/drivers/ble/FEATURE_BLE/TARGET_STM32WB/STM32Cube_FW/ble_bufsize.h

```diff
@@ -0,0 +1,161 @@
+/*****************************************************************************
+ * @file    ble_bufsize.h
+ * @author  MCD Application Team
+ * @brief   Definition of BLE stack buffers size
+ *****************************************************************************
+  * @attention
+  *
+  * <h2><center>&copy; Copyright (c) 2019 STMicroelectronics.
```

Comment:
I recall seeing other cube drivers using SPDX, not this version?

Reply:
Mmm, this is because file comes from CubeFW without any modification....

Reply:
Can you add SPDX identifiers and send also upstream?

Reply:
As I said, file comes from https://github.com/STMicroelectronics/STM32CubeWB/blob/master/Middlewares/ST/STM32_WPAN/ble/core/ble_bufsize.h
Like in https://github.com/ARMmbed/mbed-os/tree/master/targets/TARGET_STM/TARGET_STM32WB/STM32Cube_FW
I prefer to avoid any modification to ease driver updates when there is a new ST package

Reply:
Travis fails as SPDX is required for all new files :(

the only way to address this would be to add the driver to license check ignore.

Reply:
You should ignore all files */STM32Cube_FW/*
in targets/TARGET_STM
and in other directories like this one....

Reply:
Ok, please add them to codecheckignore list. This will make Travis green.